.prettierignore

```diff
@@ -0,0 +1,5 @@
+*
+!**/
+!*.scss
+!*.js
+js/redirect.js
```

Comment:
```suggestion
**/js/redirect.js
```

Or `_site` instead, otherwise `prettier --check` will fail if the website was built

Reply:
Ooh nice catch. Have added `_site`, no reason to check the built site.